aws-cloudfront-naked-domain-redirect/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable project {
+  type        = string
+  description = "Project for tagging and naming. See [doc](../README.md#consistent-tagging)"
+}
+
+variable env {
```

Comment:
**[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
variable "env" is declared but not used